File: testdata/fixtures/ai/dart/detect-gemini.dart
// AI usage detection: Gemini via google_generative_ai
// ruleid: detect-gemini
import 'package:google_generative_ai';
// ruleid: detect-gemini
final model = GenerativeModel('gemini-pro', 'key');
// ruleid: detect-gemini
final other = GenerativeModel('gemini-flash', 'key');
// ok: detect-gemini
import 'dart:io';
// ok: detect-gemini
final text = 'plain string';
// ok: detect-gemini
print('hello');

File: testdata/fixtures/ai/kotlin/detect-gemini.kt
// AI usage detection: Gemini via google.ai
// ruleid: detect-gemini
import com.google.ai
import kotlin.String
// ruleid: detect-gemini
val model = GenerativeModel("gemini-pro", "key")
// ruleid: detect-gemini
val other = GenerativeModel("gemini-flash", "key")
// ok: detect-gemini
import kotlin.collections
// ok: detect-gemini
val text = "plain string"
// ok: detect-gemini
val x = 1
